source/README.adoc

```diff
@@ -0,0 +1,142 @@
+
+:toc:
+:toc-placement!:
+= RabbitMQ Source
+
+The RabbitMQ source translates messages on a RabbitMQ exchange to CloudEvents
+over HTTP for use with Knative Eventing. The Source can bind to an existing
+exchange or create a new one, as needed.
+
+toc::[]
+
+
+== Published Events
+
+All messages received by the Source will be published with the following schema:
+
+.Table Event Attributes
+|===
+| Attribute | Value | Notes
+
+| `type` | `dev.knative.rabbitmq.event` |
+| `source` | `/apis/v1/namespace/*$NS*/rabbitmqsources/*$NAME*#*$TOPIC*`
+   | `NS`, `NAME` and `TOPIC` are derived from the Source configuration.
+| `id` | a unique id | This uses the MessageId if available, and a UUID otherwise.
+| `subject` | the message's id | empty string if no message ID is present.
+| `datacontenttype` | `application/json` | Currently static.
+| `key` | the message's id | empty string if no message ID is present.
+|===
+
+The event's payload will be set to the data content of the message.
```

Comment:
```suggestion
The payload of the event is set to the data content of the message.
```
Use present tense instead of future consistently, and avoid possessives, e.g. "event's" where possible.

Reply:
Done.